### DISCUSSION TITLE
hi noob here the TS-HTGL/R5 is supported by this project?

### DISCUSSION THREAD

**Discussion by MartinEtcheverry** [1000001101000/Debian_on_Buffalo#213] [Q&A]
Is the TS-HTGL/R5 is supported? if it is, what is the name of the files that i have to download? uImage-88f6281.buffalo.tsxel and initrd.buffalo

thank you for your time!
have a great day!

**Comment by 1000001101000** [comment]
it will be the ts2pro file + that initrd

**Reply by MartinEtcheverry** [reply]
thanks a lot for your response !!!! i will try it !
